.github/workflows/updated-at-field.yml

```
@@ -0,0 +1,26 @@
+name: Update Index
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  spell:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.DOKA_BOT_ACCESS_TOKEN }}
```

```suggestion
    steps:
      - uses: actions/checkout@v2
        with:
          token: ${{ secrets.DOKA_BOT_ACCESS_TOKEN }}
      - uses: actions/setup-node@v2
        with:
          node-version: '14'
```

Удивился, что мы используем `node`, но нигде не делаем `actions/setup-node`, что как будто необходимо для такого. Плюс я убрал `strategy.matrix.node-version` просто в строку.